### ISSUE TITLE
Vercel deployment: Cn we reuse the other build to upload ?

### ISSUE THREAD

**Issue by Carreau** [jupyter/papyri#225]
For the Vercel deployment, can we use the html built in `python-package.yml` instead of rebuilding.

**Comment by aktech** [comment]
Yes, indeed. We can upload that as GitHub Artifact and use that in the preview step.
